pkg/controller/v2/controller.go

```diff
@@ -1,11 +1,12 @@
 // SPDX-FileCopyrightText: 2023-present Intel Corporation
-//
+// 
```

Comment:
Jekins is failing due to formatting in pkg/controller/v2/controller.go
```suggestion
//
```